Complex vector spaces (#1692)
UniMath/SymmetryBookFormalization: src/real-numbers/multiplication-nonzero-real-numbers.lagda.md

mul-nonzero-ℝ (x , x#0) (y , y#0) =
  ( x *ℝ y , is-nonzero-mul-ℝ x#0 y#0)
```

## Properties

> ### If the product of two real numbers is nonzero, they are both nonzero

Reviewer: ```suggestion
### If the product of two real numbers is nonzero, each factor is nonzero
```
